README.md

```diff
@@ -138,6 +120,21 @@ Support for the following operating systems **have been removed**:
 * Windows XP
 * Mac OS X 10.6 through 10.9
 
-**Q: What version of Python does funfuzz require?**
+**Q:** What version of Python does domfuzz require?
 
 **A:** We recommend the Python 2.7.x series. There is no support for Python3 yet.
+
+**Q:** How do I reproduce a crash from a test case?
+
+**A:** The domfuzz way:
+```bash
+ python ./domfuzz/dom/automation/domInteresting.py ~/build ~/test_case.html
+ ```
```

Comment:
You can format code without \`\`\` by surrounding with blank lines and indenting 4 spaces.